### PR TITLE
Feat: Add Dockerfile for ONCOCNV

### PR DESCRIPTION
**Pull request by marrip**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: oncocnv/v7.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:20.04 AS download
+LABEL base_image="ubuntu:20.04" \
```
Comment (osallou) [inline oncocnv/v7.0/Dockerfile]: labels should be in final image, not intermediate one used to download file

File: oncocnv/v7.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      version="1" \
+      software="ONCOCNV" \
+      software.version="v7.0" \
+      about.summary="ONCOCNV - a package to detect copy number changes in Deep Sequencing data" \
+      about.home="https://github.com/BoevaLab/ONCOCNV" \
+      about.documentation="https://github.com/BoevaLab/ONCOCNV/blob/master/README.md" \
+      about.license="none" \
```
Comment (osallou) [inline oncocnv/v7.0/Dockerfile]: software must have a license
no license does not mean open source nor free to share

**Review by osallou — CHANGES_REQUESTED** [review]

File: oncocnv/v7.0/Dockerfile (hunk 1 above)
Comment (marrip) [reply oncocnv/v7.0/Dockerfile]: done

File: oncocnv/v7.0/Dockerfile (hunk 2 above)
Comment (marrip) [reply oncocnv/v7.0/Dockerfile]: found it and added it to the label

File: oncocnv/v7.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+FROM ubuntu:20.04
+LABEL base_image="ubuntu:20.04" \
+      version="1" \
+      software="ONCOCNV" \
+      software.version="v7.0" \
+      about.summary="ONCOCNV - a package to detect copy number changes in Deep Sequencing data" \
+      about.home="https://github.com/BoevaLab/ONCOCNV" \
+      about.documentation="https://github.com/BoevaLab/ONCOCNV/blob/master/README.md" \
+      about.license="GNU General Public License v3.0" \
+      extra.identifiers.biotools="https://bio.tools/ONCOCNV" \
```
Comment (osallou) [inline oncocnv/v7.0/Dockerfile]: Should only set biotools id, not url  ie oncocnv here

Comment (marrip) [reply oncocnv/v7.0/Dockerfile]: does that mean `biotools:oncocnv` is the ID?

Comment (osallou) [reply oncocnv/v7.0/Dockerfile]: no, id is oncocnv

**Comment by osallou** [comment]
@marrip merged and thanks for contribution

container will be available soon after build (tagged, no *latest* available)

**Pull request merged**